app/service-map.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: page
+title: Service map
```

Comment:
I'd argue this wasn't a 'service map' as most people would expect. Maybe call it the 'Teacher services diagram'?

Reply:
Map and diagram feel fairly synonymous to me? (eg "tube map" / "mind map"). It's probably technically a diagram, but I think most people call it a map? 

How about "Teacher services map"?

Reply:
Yeah, that works